.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "mdai-gateway (GoLand)",
+  "build": { "dockerfile": "Dockerfile", "context": ".." },
+  "workspaceFolder": "/workspaces/mdai-gateway",
+
+  "mounts": [
```

Comment:
Its a good idea to add:
```
"source=${localEnv:HOME}/.ssh,target=/home/vscode/.ssh,type=bind,consistency=cached"
```
So that we can do all the git stuff from within the devcontainer. I can't recall if devcontainer add this mount for you automagically or not, but I always add it explicitly just in case